experiments.md

```diff
@@ -16,7 +16,20 @@ export DATA_DIR=/absolute/path/to/data  # where you want to put scale=100/partit
 docker run --rm \
     -v ${DATA_DIR}:/dataset/tpch:rw \
     --name dataset deepola-wake:sigmod2023-data \
-    bash data-gen.sh 100 100 /dataset/tpch
+    bash data-gen.sh 100 100 /dataset/tpch && \
+    python3 convert-to-parquet.py /dataset/tpch/scale=100/partition=100/tbl
```

Comment:
@mIXs222 Do you know of a way to run these two commands together? Seems like the `&&` is separating out the `docker run` and `python3 convert-to-parquet` as two separate commands and thus giving error.

Reply:
I think you can put them in quote, like `"sleep 3 && echo hello"`